src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -158,7 +157,7 @@ Status CoreWorkerMemoryStore::Put(const ObjectID &object_id, const RayObject &ob
 
     auto iter = objects_.find(object_id);
     if (iter != objects_.end()) {
-      return Status::ObjectExists("object already exists in the memory store");
+      return Status::OK();  // Object already exists in the store, which is fine.
```

Comment:
Should we check for equality?

Reply:
Yeah, I'm pretty sure there are no calls that depended on the previous behaviour (since the signature is changed, and I manually fixed up all the broken call sites).